signalfx-java/pom.xml

```diff
@@ -80,6 +80,12 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-annotations</artifactId>
     </dependency>
+    <dependency>
+      <!-- Temp work-around until httpclient upgrades its codec version -->
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.15</version>
```

Comment:
you don't need to add the version number here, having it in the parent pom is enough. None of the other dependencies in this file add a version number. Another option is to not add this dependency here at all, having it in parent is sufficient.